CRM/Case/Form/Activity.php

```diff
@@ -279,7 +279,7 @@ public function buildQuickForm() {
       // Is that really a big problem?
       $this->_encounterMedium = CRM_Core_DAO::getFieldValue('CRM_Activity_DAO_Activity', $this->_activityId, 'medium_id');
       if (!array_key_exists($this->_encounterMedium, $encounterMediums)) {
-        $encounterMediums[$this->_encounterMedium] = CRM_Core_OptionGroup::getLabel('encounter_medium', $this->_encounterMedium, FALSE);
+        $encounterMediums[$this->_encounterMedium] = CRM_Core_PseudoConstant::getLabel('CRM_Activity_BAO_Activity', 'encounter_medium', $this->_encounterMedium);
```

Comment:
I believe this is incorrect and should be 'medium_id' instead of 'encounter_medium'

Reply:
Yes